src/test/java/com/github/matidominati/medicalclinic/service/PatientServiceTest.java

```diff
@@ -178,19 +185,18 @@ void updatePatient_PatientIdChanged_PatientNotUpdated() {
 
     @Test
     void changePassword_EmailCorrect_PasswordChanged() {
-        Patient patientToChangePassword = new Patient(1,"patient.patient@gmail.com", "aa", "123", "bb",
+        Patient patientOriginal = new Patient(1,"patient.patient@gmail.com", "aa", "123", "bb",
                 "cc", "124", LocalDate.of(1999, 2, 1));
-        Patient updatedPatient = new Patient(1,"patient.patient@gmail.com", "patient1", "123", "bb",
+        Patient patientUpdated = new Patient(1,"patient.patient@gmail.com", "patient1", "123", "bb",
                 "cc", "124", LocalDate.of(1999, 2, 1));
 
-        when(patientRepository.findByEmail(patientToChangePassword.getEmail())).thenReturn(Optional.of(patientToChangePassword));
-        when(patientValidator.checkPatientPassword(updatedPatient)).thenReturn(true);
-
-        patientService.changePassword(patientToChangePassword.getEmail(), updatedPatient);
+        when(patientRepository.findByEmail(patientOriginal.getEmail())).thenReturn(Optional.of(patientOriginal));
+        when(patientValidator.checkPatientPassword(patientUpdated)).thenReturn(true);
```

Comment:
patientValidator.checkPatientPassword -> jesli metoda zwraca true lub false to nazywajmy ja tak zeby to sugerowalo ze zwraca 
Przykladowo:
isPatientPasswordValid